servant/src/Servant/API/Delete.hs

```diff
@@ -12,9 +12,7 @@ import Data.Typeable ( Typeable )
 --
 -- >>>            -- DELETE /books/:isbn
 -- >>> type MyApi = "books" :> Capture "isbn" Text :> Delete
-data Delete (contentTypes :: [*]) a
-  deriving Typeable
-
+type Delete (contentTypes :: [*]) a = HttpMethod "DELETE" 200 contentTypes a
```

Comment:
The doctest should say `Delete <content-type> ()` (our mistake - only now are the doctests failing here because previously it'd still compile)